### PR TITLE
Display search results in cards

### PR DESCRIPTION
**Pull request by emilyzhaoo**

### CODE REVIEW

File: planet/src/main/webapp/places.js [hunk 1]
```diff
@@ -101,27 +100,37 @@ function initMap() {
...
-/** List results of nearby search */
+/** List results of nearby search on page*/
 function listResults(results) {
     const places = document.getElementById('places');
 
+    // Displays search results on page
     for (var i = 0; i < results.length; i++) {
-        places.innerText = places.innerText + <br /> + results[i]; 
+        // Create materialize card for each result by creating dom element in HTML
+        var div1 = document.createElement("div");
```
Comment (azhang72) [inline planet/src/main/webapp/places.js]: nit: s/var/let.

**Review by azhang72 — APPROVED** [review]
lgtm with a nit

File: planet/src/main/webapp/places.js [hunk 2]
```diff
@@ -101,27 +100,37 @@ function initMap() {
...
+/** List results of nearby search on page*/
 function listResults(results) {
     const places = document.getElementById('places');
 
-    for (var i = 0; i < results.length; i++) {
-        places.innerText = places.innerText + <br /> + results[i]; 
+    // Displays search results on page
+    for (let i = 0; i < results.length; i++) {
+        // Create materialize card for each result by creating dom element in HTML
+        let div1 = document.createElement("div");
```
Comment (EkkiD) [inline planet/src/main/webapp/places.js:112]: Why do we need 3 divs? Is there materialize documentation we can link to here?

**Review by EkkiD — APPROVED** [review]
LGTM with nit

File: planet/src/main/webapp/places.js (hunk 2 above)
Comment (emilyzhaoo) [reply planet/src/main/webapp/places.js:112]: Is there a way to do that? I manually created the materialize card (which required 3 divs)

Comment (EkkiD) [reply planet/src/main/webapp/places.js:112]: I meant a link in the comment to the documentation that explains what the 3 divs are. Maybe this one? https://materializecss.com/cards.html#horizontal

**Pull request merged**